python/version.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in project root for information.
+
+import re
+
```

Comment:
I think we'll want to just add this code as extra files in the deep learning python code so that the codegen system naturally fits in with this. If the setup.py that gets generated is wrong in some way for this project, perhaps we should use this as a first example in codegen of a custom or overriding setup.py for a project so that future developers can benefit from this exploration.